cfme/tests/cloud/test_security_groups.py

```diff
@@ -0,0 +1,59 @@
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.wait import TimedOutError
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider([OpenStackProvider], scope="module")
+]
+
+
+@pytest.yield_fixture(scope='module')
+def sec_group(appliance, provider):
+    security_group = appliance.collections.security_group
+    try:
+        sec_groups = security_group.create(name=fauxfactory.gen_alphanumeric(),
+                                           description=fauxfactory.gen_alphanumeric(),
+                                           provider=provider,
+                                           wait=True
+                                           )
+    except TimedOutError:
+        pytest.fail('Timed out creating Security Groups')
+    yield sec_groups
+    sec_groups.delete(wait=True)
```

Comment:
**REQUIRED**
You are deleting the security group in the test below, so here an error will occur. In order to avoid that check the existence of the security group:
```python
if sec_groups.exists:
    sec_groups.delete(wait=True)
```

Reply:
Oops forgot to add this.